docs/reST/ref/mask.rst

```diff
@@ -45,8 +45,31 @@ Starting from pygame 1.9.5 masks with width or height 0 are supported.
 
 .. class:: Mask
 
-   | :sl:`pygame object for representing 2d bitmasks`
-   | :sg:`Mask((width, height)) -> Mask`
+   | :sl:`pygame object for representing 2D bitmasks`
+   | :sg:`Mask(size=(width, height)[, fill=False]) -> Mask`
+
+   A ``Mask`` object is used to represent a 2D bitmask. Each bit in
+   the mask represents a pixel. 1 is used to indicate a set bit and 0 is used
+   to indicate an unset bit. Set bits in a mask can be used to detect
+   collisions with other masks and their set bits.
+
+   A filled mask has all of its bits set to 1, conversely an unfilled/cleared
+   mask has all of its bits set to 0. Masks can be created unfilled (default)
+   or filled by using the ``fill`` parameter. Masks can also be cleared or
+   filled using the :func:`pygame.mask.Mask.clear()` and
+   :func:`pygame.mask.Mask.fill()` methods respectively. Individual bits can
+   be accessed using the :func:`pygame.mask.Mask.get_at()` and
+   :func:`pygame.mask.Mask.set_at()` methods.
+
+   :param list|tuple size: sequence of 2 ints indicating the width
```

Comment:
I think the correct syntax is `:param list or tuple size:`, according to http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html

Reply:
Thanks for the suggestions. The `param` field doesn't seem to work using multiple words for the type, so I had to use the `type` field. Also, I decided to avoid using `sequence` because it doesn't just mean a tuple or list (e.g. string). I've updated the code accordingly.